packages/cli/src/commands/rewards/show.ts

```diff
@@ -253,9 +253,9 @@ export default class Show extends BaseCommand {
         {
           groupName: { get: (e) => e.group.name },
           group: { get: (e) => e.group.address },
-          groupPayment: { get: (e) => e.groupPayment.toFixed() },
+          groupPayment: { get: (e) => e.groupPayment.toFixed(0) },
           validator: { get: (e) => e.validator.address },
-          validatorScore: { get: (e) => e.validator.score.toFixed() },
+          validatorScore: { get: (e) => e.validator.score.toFixed(0) },
```

Comment:
don't think we want to fix this for validatorScore. this should be essentially a fraction of 1

Reply:
hi @aslawson 
changed m find below the tests result:

./run rewards:show --voter 0xc64DF5Be250264bf2888591D87cdeB13BFADC501 --privateKey 7a0970e7181b9003581c8870a7631addc6a7001c4ae896cd1d0363e528cfaa0b --epochs=1 --estimate --csv

Voter rewards:
Address,Addresspayment,Group,Averagevalidatorscore,Epochnumber
0xc64DF5Be250264bf2888591D87cdeB13BFADC501,1507771884444893904,0xc64DF5Be250264bf2888591D87cdeB13BFADC501,1,331

./run rewards:show --privateKey 7a0970e7181b9003581c8870a7631addc6a7001c4ae896cd1d0363e528cfaa0b --epochs=1 --csv

Group voter rewards:
Groupname,Group,Groupvoterpayment,Averagevalidatorscore,Epochnumber
Moonlet.io,0x8851F4852ce427191Dc8D9065d720619889e3260,418834222630641240795,1,331
dynamic,0xD018838832f6112dE72EE6CC9967C56B333a0d1C,413107272308517978371,1,331
huglester,0x614B7654ba0cC6000ABe526779911b70C1F7125A,417649347211990615439,1,331
nonce,0xC7d5409fEe80B3Ac37Dbc111664dC511a5982469,414619748941300716439,1,331
Projecttent,0x3D451dd723797b3DE938C5B22412032B6452591A,417108510401954209591,1,331
Kytzu Group,0xc64DF5Be250264bf2888591D87cdeB13BFADC501,417287626884042946513,1,331
Figment Networks,0x01b2b83fDf26aFC3Ca7062C35Bc68c8DdE56dB04,1229669969949383165865,1,331
syncnode_group,0x82f8Bcf96f24BA60Ef041D192c7CE04C907E2fb8,422266233195409792502,1,331
DSRV CeloWhale.com,0xF83C93ea360B66DDCD532960304948B1c10786a1,507961731077603274305,1,331
Simply Staking,0xAcdf897493A6000dbe256791E8A2beCbb405FD4F,429049001909149979994,1,331
Easy2Stake Group,0x061E9958028dcAa66fd8B255AD95194203b6c4Da,417152598474124774933,1,331
sparkpool-group,0xEEFCfDFc8F5CeD9799A13EcA58DE2ba7534eAB92,419035986589659650927,1,331
Polychain Labs,0x47b2dB6af05a55d42Ed0F3731735F9479ABF0673,1714121143488859472202,1,331
missionquorum-group,0xA432da0Ed5A2c15cbc681227cCec3b375908FdCB,1263670554173567933361,1,331
cnstnt.xyz,0x21FB4411FA5828344c2788aB07D4cc12a12571b9,958893826073066686622,1,331
,0x7dbB949346Aeda9233218831455d57Eb8344Af82,1211271578192427022693,1,331
,0x9eB1D96B5f0bdD3114eA1461182c688178ae6c4c,910059317401715982800,1,331
Cypher Core,0x580F29aB04b7ED4E5aA86584D38c9317d9CF0358,417719735197621229554,1,331
ChainLayer,0x7194DFE766a92308880A943fD70F31c8E7c50e66,463005974946237443941,1,331
a16z,0x4dA92da1afbf103F2B52DD29326e34C98cA1E78c,1137001994300472553604,1,331
StakesStoneGroup,0x3DcF2ED8dC84a63FfD2bfDc3CDb2fA0B1aeAfE5c,418488416461903772353,1,331
Staking Fund,0xFaa0b40645Ee5E8D9AE38393e8cDF8e5baA71d13,418331984827908950840,1,331
Bison Trails,0xDaDBd6Cfb29b054aDC9c4c2Ef0F21f0BBdb44871,1270438881633365409458,1,331
“Gossip”,0x8954661e743c60C966E2ff6002B514126bb1cFe2,354906709328563678391,1,331
The Passive Trust,0xB87f2354E34B26ba6406Ac60EA99DCD8cd5e63Bf,540914843194367367876,1,331
gunray,0x5402172E972b31Fc9F0383F53f45823Ab5037379,416141000204946360577,1,331
Tango,0x98c62d86634bB0E3f818EFC550e2F33369Eae7F3,619430921811014294363,1,331
Nodito Labs,0x2c2B0f71d59B546B2CAfd222696589c13C3c325C,918600837350619477402,1,331
Anchorage,0x35aE10F412503aBcf9275133613E8df7f56E72Be,2193636496918659924300,1,331
CoinList,0x6f8e6ff69935F000B09d182A4739342fAEaF0c77,962019738090177354761,1,331
Manta Labs,0xbf55dF76204f00AcF296F76cF4Aaf86A866a5eb0,920569378023877531770,1,331
Bitcandy 🍬,0x34649AdA2cB44D851a2103Feaa8922DedDABfc1c,498479643651934476556,1,331
Bi23 Labs,0x07fa1874ad4655AD0C763a7876503509be11e29E,435513868858154146791,1,331
HashQuark,0xD19FB36B7F433fe13820767ef6d0E26FDbaB68CC,411837065775367428043,1,331
Celo.Works,0x3C86b6A27a074c1c4CC904d8808A1C33078DB4E6,444112249509952348497,1,331
,0x89d5bd54C43dDd10905A030DE6Ff02EbB6c51654,906001834150335571487,1,331
Chorus One,0x81cef0668e15639D0b101bdc3067699309D73BED,420630241380481876731,1,331
Chainflow,0x249a6B2B260000b08f50A2480e2d703bAf02E8BE,412673410720218743407,1,331
Blockdaemon,0xCc4B2Bcbbc9639ef1E91F47acfD12Bd131525e79,500634135886115034841,1,331
Perfect Stake,0x6CF4bB9ff947610944c6D8E0E5Ea26B1dEa73196,417147421729259718667,1,331
Ryabina | Celomap.io,0x8493BD3De67AC341D4cC11531f96a1A2cdBf29aD,422634839234698739762,1,331
Tessellated Geometry Alliance,0x0339Df3FE4f5ccC864EAE8491E5c8AEc4611A631,419649622138113787409,1,331
SlavaMo,0x602B65795BCc64b2fb329AC004236E194f077158,417688842283672640232,1,331
The Celo Group,0x81AE1C73A326325216E25ff1af9EA3871195036E,421686622190630082856,1,331
nga_group,0xc24baeac0Fd189637112B7e33d22FfF2730aF993,416116432532088996533,1,331
Masternode24.de,0x95aE59515915D6c493a846aE022F93726652b50A,416197758167723908939,1,331
AG,0x15Ed3f6b79F5Fb9Ef1D99D37314Dd626b3005F0b,419000201303982483499,1,331
moonli.me,0xc8A81D473992c7c6D3F469A8263F24914625709d,425874200939522960501,1,331
Alive29,0x0f66619058BB9675f3d394FCc2cE236a29901571,421614123801554190297,1,331
chainvibes,0xD72Ed2e3db984bAC3bB351FE652200dE527eFfcf,423749584566883940862,1,331
Virtual Hive,0x71dcC67baEECd9308e341359f70B782D9C3565b8,226649746645057532024,1,331
HappyCelo,0x481EAdE762d6D0b49580189B78709c9347b395bf,423809559295092726612,1,331
Censusworks,0x0d4f8cEA48cDaDEAe345431577a64983c0535B12,546129717863202896492,1,331
Validator.Capital,0xB33e9e01E561a1Da60f7Cb42508500e571afb6Eb,434651364751091522622,1,331
wotrust,0x7C75B0B81A54359E9dCCDa9cb663ca2e3De6B710,624694131201886673368,1,331
keyko.io,0x2fd49E97262D505Fd76BB6E0e06eC10e1fd54589,454689332445343735341,1,331
PretoriaResearchLab.io,0xddaa60B6d803674bBc434F1C2B261CeB67C2fd7c,459231321429046116277,1,331
Usopp | usopp.club,0x067e453918f2c44D937b05a7eE9DBFB804C54ADd,441061366419838985009,1,331
Goods & Services,0x8580dB53C3ebC56230662B771ceF2707E92Ef83A,423214704214624276671,1,331
celvaly,0xd25c6a9FEf4744E8d4F90Bf6bdFAF7686909d799,420145388514731097313,1,331
swiftstaking,0xE141831c2c1198d79B9Ff61cD97C3bAca7F071E0,421266187983889622506,1,331
TDlabs | Staked.us,0x8a12caB622B8093208931fA008D12D6Ba5AF47E4,618404282278775461999,1,331
Conclave,0xAF2b000bAed202fA7de0d9DCeB6F6612De348011,444360103320313899774,1,331
ZanshinDojo.org,0x8eB004daD9397B8f23E1279905c584920000756D,440212822562587875621,1,331
AUDIT.one,0xc764293De02189e248a8040a3D1b04EeC599Aa31,182888006825071272812,0,331
TeroLabs,0x0E912990472bfe5948B5b7333Ecf528D5dA887cF,182887450152455657549,0,331
StakeService.com,0xb35Be22BccB0dB9dC62967dcF15fEB97C20f854e,179955971252127033510,0,331
YYYYYYYYYYYYY,0xE075ba4b1dCAF75513118d7aA08A057c658842c9,428778055220280090607,1,331
deDataLabs,0x1D083BECFAac301B4B845d0E5B80A9Da5F6DE946,351222398294603422881,1,331

Validator rewards:
Validatorname,Validator,Validatorpayment,Validatorscore,Group,Epochnumber
MoonletV1,0x399b3D48Fdcd668d274afc6F4be05475A325C6c6,164293699923731639697,1,0x8851F4852ce427191Dc8D9065d720619889e3260,331
dynamic,0x3BEbD7Ed4cA0ce7BFd69fC6aeF31dDD21C460158,166160112413366207295,1,0xD018838832f6112dE72EE6CC9967C56B333a0d1C,331
huglester,0x5e69Cca114a77AB0a5804108FAF0cD1E1C802A5e,166263247365857989501,1,0x614B7654ba0cC6000ABe526779911b70C1F7125A,331
nonce,0x1819b44553Fa39A983D1c92238E1F18AeE30eF51,166321674213493163260,1,0xC7d5409fEe80B3Ac37Dbc111664dC511a5982469,331
Projecttent 1,0xCDd88f80ed52AfEc9D670f5a6Da8940EEe716f45,163682487972487441341,1,0x3D451dd723797b3DE938C5B22412032B6452591A,331
kytzu,0xbBC724bdf633287E8bC18aC45800e2eD13877a73,163328690730383317508,1,0xc64DF5Be250264bf2888591D87cdeB13BFADC501,331
Figment Networks 1,0x0e47ea88480788289bDa37D8Ae0C2CD680623cfA,184771153435474475457,1,0x01b2b83fDf26aFC3Ca7062C35Bc68c8DdE56dB04,331
syncnode,0xD5Fb8A8E6E81BC77d7D807c05b5b8aEAe7c2a458,184706018924880165807,1,0x82f8Bcf96f24BA60Ef041D192c7CE04C907E2fb8,331
DSRV - CeloWhale.com - 1,0x588eD7dEefF8DF7c00FF187C398936b101583ec7,142055827407105018051,1,0xF83C93ea360B66DDCD532960304948B1c10786a1,331
Simply Staking,0xc2023E0D13F53d181f29B99D829CD75628337D15,182711541807631095820,1,0xAcdf897493A6000dbe256791E8A2beCbb405FD4F,331
Easy 2 Stake,0x1Dd301fEaa72F4C07b3ca56ce0C27e98E66B77Bd,165175376486947997661,1,0x061E9958028dcAa66fd8B255AD95194203b6c4Da,331
sparkpool-v1,0xA87710B8DC870483a2bE526EBca9F5048720eDce,166288513129141330747,1,0xEEFCfDFc8F5CeD9799A13EcA58DE2ba7534eAB92,331
Polychain Labs Classic,0x1d511366FEB1ed330925B2D51645bb5c607EA6D4,0,1,0x47b2dB6af05a55d42Ed0F3731735F9479ABF0673,331
Polychain Labs Cybernetic,0x5282F82f82E9a74E186b8Ee18cc76808b78DC383,0,1,0x47b2dB6af05a55d42Ed0F3731735F9479ABF0673,331
missionquorum-0,0x39b01620ecdEB8347270ee78dC962b0EA583b57a,0,1,0xA432da0Ed5A2c15cbc681227cCec3b375908FdCB,331
Polychain Labs Rhodium,0xAB0C16BBd78D6b9F4503D81D859A3FFB64486F9A,0,1,0x47b2dB6af05a55d42Ed0F3731735F9479ABF0673,331
Polychain Labs Seraph,0xACF0E11ff39EaD797475a1694FCca94A0e94784F,0,1,0x47b2dB6af05a55d42Ed0F3731735F9479ABF0673,331
cnstnt.xyz/rc1/validator-1,0x8d1A5eF84252586f0f841a9fCF4E31f23c4c21E0,0,1,0x21FB4411FA5828344c2788aB07D4cc12a12571b9,331
Mimetic Polyalloy,0x1E554ac038114C0A69834Be928FB95e5aaF18950,0,1,0x47b2dB6af05a55d42Ed0F3731735F9479ABF0673,331
,0x822D6b9BBA2A3E460f3aB45fAAED98993522A8F0,0,1,0x7dbB949346Aeda9233218831455d57Eb8344Af82,331
cnstnt.xyz/rc1/validator-2,0xDF9121AFFb220e9095C875a79c3D1fA3EBEE4105,0,1,0x21FB4411FA5828344c2788aB07D4cc12a12571b9,331
,0x5423c03551641336b3D5886f85F882FD2BD5DB7E,0,1,0x7dbB949346Aeda9233218831455d57Eb8344Af82,331
,0x6964A0C39FA7f63C59F7dC77aC1802f24E2649Aa,0,1,0x7dbB949346Aeda9233218831455d57Eb8344Af82,331
,0xDd431a27ADce5B957FE2FE877a65F0e6304D68a0,0,1,0x9eB1D96B5f0bdD3114eA1461182c688178ae6c4c,331
,0xdA71A321C75A0Fe76073384bC52f6D583EAec517,0,1,0x9eB1D96B5f0bdD3114eA1461182c688178ae6c4c,331
Cypher Core,0x4e957bBC61AFDab63263fFdbAF201F6f596E3e2D,165440059756859307352,1,0x580F29aB04b7ED4E5aA86584D38c9317d9CF0358,331
ChainLayer1,0x4FC4EA624db2E4A1d6195a03744D505CbcD9431b,166334032465697571941,1,0x7194DFE766a92308880A943fD70F31c8E7c50e66,331
a16z - 1,0xb2728EDcc9c255C2D053Ea6Cae31cFd584B758D3,0,1,0x4dA92da1afbf103F2B52DD29326e34C98cA1E78c,331
a16z - 2,0xFcE0779607246eb5121a5A4192D6AC4A758C217D,0,1,0x4dA92da1afbf103F2B52DD29326e34C98cA1E78c,331
a16z - 3,0x394B846E0eCf83Ef350A2A7Ccb7Be0C6552e7cBC,0,1,0x4dA92da1afbf103F2B52DD29326e34C98cA1E78c,331
Figment Networks 2,0x90b056B00952d71bC7866c067b49f5b062098ac9,171735878060953319460,1,0x01b2b83fDf26aFC3Ca7062C35Bc68c8DdE56dB04,331
Figment Networks 3,0xDdc60B465c204Aa7358cE4f009D2dBC65af5C4B4,170605052274551422886,1,0x01b2b83fDf26aFC3Ca7062C35Bc68c8DdE56dB04,331
StakesStone,0x5F2E4e239bCD86e1cf97538Ea7D5E2Ec639a3Bc4,165216552125303334645,1,0x3DcF2ED8dC84a63FfD2bfDc3CDb2fA0B1aeAfE5c,331
Staking Fund #1,0x7b3859147B1C173FfF7182E58A4Ec81C1d287224,184859653863180790921,1,0xFaa0b40645Ee5E8D9AE38393e8cDF8e5baA71d13,331
Bison Trails 1,0x089B1654325CcE14fb3e2dABCa2803bc489e83e7,0,1,0xDaDBd6Cfb29b054aDC9c4c2Ef0F21f0BBdb44871,331
,0x2DF505a1f281cd3ba10baA093fa89909C75362b5,165785157428250155216,1,0x8954661e743c60C966E2ff6002B514126bb1cFe2,331
The Passive Trust - V1,0x29c9CA0186650063ba6fcaC3a952613B11d6bC36,110885076585628023507,1,0xB87f2354E34B26ba6406Ac60EA99DCD8cd5e63Bf,331
gunray,0xe5F947368C986DFe7a025B9B35deE3e5AC3a330e,166362300576952190161,1,0x5402172E972b31Fc9F0383F53f45823Ab5037379,331
yira-yira,0xeD32008dB382d078bABB0eB6F91Df0207d909899,158874850634615330435,1,0x98c62d86634bB0E3f818EFC550e2F33369Eae7F3,331
Nodito Labs 1,0xaB6eeE9dA588d38B43B1d2ea404e824fA2bF382c,138644645753799604944,1,0x2c2B0f71d59B546B2CAfd222696589c13C3c325C,331
,0x43fbd2B45Ef5C042F821D24E14933C570e05Ef78,0,1,0x35aE10F412503aBcf9275133613E8df7f56E72Be,331
,0x10bCB8943B7BF9b33E4D510feB212bD20b1cE9B2,0,1,0x6f8e6ff69935F000B09d182A4739342fAEaF0c77,331
,0x88f243bc0f439549DFFFF3C8c60b82f298116C9d,0,1,0x6f8e6ff69935F000B09d182A4739342fAEaF0c77,331
,0xadBE75cBD14f92BA2D233b6b81E80D4634fAbB2a,0,1,0x35aE10F412503aBcf9275133613E8df7f56E72Be,331
,0x1F15d4210e34DfeeEc087C906b426509da1b01f1,0,1,0x35aE10F412503aBcf9275133613E8df7f56E72Be,331
,0xd6DCbBC16cE0fBc95866a881F8F1728C48181507,0,1,0x6f8e6ff69935F000B09d182A4739342fAEaF0c77,331
,0xc627459BFc70cD7B6719d5b8030C3beA7825CA7e,0,1,0xbf55dF76204f00AcF296F76cF4Aaf86A866a5eb0,331
Dark Chocolate,0x02DC997790352E12fA90eb1A937dFeF4bF8Dbbc0,0,1,0x34649AdA2cB44D851a2103Feaa8922DedDABfc1c,331
Bi23,0x997490F08B9b99dB00657e2B1BbEc180Cf804A27,166069883745612899881,1,0x07fa1874ad4655AD0C763a7876503509be11e29E,331
,0x112fF12927CD6f924d80b9Aba32E531733B602fF,0,1,0x35aE10F412503aBcf9275133613E8df7f56E72Be,331
,0x6e5a2624Ae582c96c5A12Eb168afA532a85AE829,0,1,0xbf55dF76204f00AcF296F76cF4Aaf86A866a5eb0,331
,0x758d1000fB3d37F88cE3BA880882898a3B0ca3aB,92222981334214132271,1,0xD19FB36B7F433fe13820767ef6d0E26FDbaB68CC,331
Celo.Works.v1,0x084efAedbD599eff54f00d1b57Fab2F792555e79,126640736039086077782,1,0x3C86b6A27a074c1c4CC904d8808A1C33078DB4E6,331
missionquorum-1,0x9e8A03768c2711c22bcc7999359D6655B7Cdb710,0,1,0xA432da0Ed5A2c15cbc681227cCec3b375908FdCB,331
missionquorum-2,0xdb2e80840f40033761A5cdFa3B09fE24f95a4aaa,0,1,0xA432da0Ed5A2c15cbc681227cCec3b375908FdCB,331
,0xcED2449cCd1aae8077eC0670224e663CE19C8Fc4,0,1,0x89d5bd54C43dDd10905A030DE6Ff02EbB6c51654,331
,0x8CC5De79fC6C21531577F7B3d62c8413FB3bd303,0,1,0x89d5bd54C43dDd10905A030DE6Ff02EbB6c51654,331
,0x30e0bfC25Df7642ee19Ff4fbCbE9ddeA4D1935bc,0,1,0xDaDBd6Cfb29b054aDC9c4c2Ef0F21f0BBdb44871,331
Chorus One Validator1,0x5098A28bFa2Da3183E36C009c7D63093b66441a5,170670838115216762694,1,0x81cef0668e15639D0b101bdc3067699309D73BED,331
cnstnt.xyz/rc1/validator-3,0x020e468c252FF18261b1996D25F0FfF4F830Cf7a,0,1,0x21FB4411FA5828344c2788aB07D4cc12a12571b9,331
,0xbcc71f31a5987F10Afd29F3BE7119C05A14F5bcB,54279533362264934817,1,0x249a6B2B260000b08f50A2480e2d703bAf02E8BE,331
blockdaemon-0,0x45d8170f30b0d408196Ab0e14AF47D77b18f0A38,164986260601442078567,1,0xCc4B2Bcbbc9639ef1E91F47acfD12Bd131525e79,331
Perfect_Stake.1,0x249eF45381b29D9955b365C33F202127499B873f,165475116439679197390,1,0x6CF4bB9ff947610944c6D8E0E5Ea26B1dEa73196,331
,0x48dE6c6668c24e5Bd113d5C46d813C50fE2204D1,0,1,0x35aE10F412503aBcf9275133613E8df7f56E72Be,331
warfollowsme | Celomap.io,0x1AEF9bf15de73aDBA318FC8aEa3D2f45d120CC8A,181851163153549075742,1,0x8493BD3De67AC341D4cC11531f96a1A2cdBf29aD,331
,0xC3957f7848d6b09DCb29d784036B0A302555E75b,0,1,0xDaDBd6Cfb29b054aDC9c4c2Ef0F21f0BBdb44871,331
,0x31cAF587f96B7C6fC8f33eB07ad854A21Eec64D9,0,1,0xDaDBd6Cfb29b054aDC9c4c2Ef0F21f0BBdb44871,331
LetzBake! V1,0xAb2fDf3F4802A542754915E04c33a9596B164F31,135806003545955938482,1,0x2c2B0f71d59B546B2CAfd222696589c13C3c325C,331
Tessellated Geometry,0x44a7Af700ACa1B24ECdB818dF4D202772D606Cb4,184855073974868995521,1,0x0339Df3FE4f5ccC864EAE8491E5c8AEc4611A631,331
SlavaMo,0xB29438f402da218180bBa79fbc0fAdD3e9E151aa,0,1,0x602B65795BCc64b2fb329AC004236E194f077158,331
NewroadNetwork,0xFE38631E7E164D20754387F7165B9C5b427dA17B,175616617179380696451,1,0x81AE1C73A326325216E25ff1af9EA3871195036E,331
nga_validator,0x4D540eC31F9CC5b1d10a329EB4Eddf52CB3dC000,91777150008929494171,1,0xc24baeac0Fd189637112B7e33d22FfF2730aF993,331
🔱Masternode24,0x9C542c0B5aF5e04E2D1e67cf1fC176E91bC18dA0,91832362736350164009,1,0x95aE59515915D6c493a846aE022F93726652b50A,331
AGx1,0x943e9a5c750adcc467575F4fE81bf6b1C3A2aeD7,147684474929909343891,1,0x15Ed3f6b79F5Fb9Ef1D99D37314Dd626b3005F0b,331
moonli.me,0xDdbca691842FD452d8F53575FdFdC2314eb3AD57,184859653863180790921,1,0xc8A81D473992c7c6D3F469A8263F24914625709d,331
Alive29x1,0x28B5f40C3f5883021408Ce79F20FCf282E873236,91390085124185585530,1,0x0f66619058BB9675f3d394FCc2cE236a29901571,331
chainvibes,0x00A9e8869e40e038D0FaE63D825a732e6FfF48d1,92428884682358009852,1,0xD72Ed2e3db984bAC3bB351FE652200dE527eFfcf,331
Carnica,0xe51662a4448cc9994Df7B28E4F0b38D678958B0D,166204532460582145666,1,0x71dcC67baEECd9308e341359f70B782D9C3565b8,331
,0xaC6492D439b3b812e822d1e975338f48bDD0B1f2,0,1,0xbf55dF76204f00AcF296F76cF4Aaf86A866a5eb0,331
HappyCelo-val-1,0xF0b25502595911b64de75F453b34557C1b2c4760,184710226431443067065,1,0x481EAdE762d6D0b49580189B78709c9347b395bf,331
censusworks.0,0x2bDc5CcdA08A7F821ae0Df72B5Fda60Cd58d6353,0,1,0x0d4f8cEA48cDaDEAe345431577a64983c0535B12,331
Validator.Capital1,0x0F412760759a2fAD4f07ceE36bfdAa218814DE89,0,1,0xB33e9e01E561a1Da60f7Cb42508500e571afb6Eb,331
wotrust1,0x6ac7E3D0BDeb78ddcF3d9b15208B50C08271C213,0,1,0x7C75B0B81A54359E9dCCDa9cb663ca2e3De6B710,331
Keyko-1,0x233c06EDc757003cA4ec078012F517F21A24C55c,166373669061452563275,1,0x2fd49E97262D505Fd76BB6E0e06eC10e1fd54589,331
Pretoria Validator A,0xe4036C56B0026eb86f058c77c55E73e9d65d2EA0,166373688401395606736,1,0xddaa60B6d803674bBc434F1C2B261CeB67C2fd7c,331
,0x5F49e61f127D315ECDE1702430EC991DAaA9C3bF,91846762011568475265,1,0x067e453918f2c44D937b05a7eE9DBFB804C54ADd,331
G&S 1 ,0xF2aADf8fF2Ad7e5C1efb0d45Cb622618BA6636A7,166373638808277336192,1,0x8580dB53C3ebC56230662B771ceF2707E92Ef83A,331
celvaly-1,0x989F3F2684F96B8cDeC308B4E7538A5a062890f0,166369297744088030601,1,0xd25c6a9FEf4744E8d4F90Bf6bdFAF7686909d799,331
wotrust2,0x206296BeffFB68fCE658535D443c732892162005,0,1,0x7C75B0B81A54359E9dCCDa9cb663ca2e3De6B710,331
swiftstaking,0x77eCf3c7AC7082344Edfee10e41Edb72fA3D1951,181289600338829254992,1,0xE141831c2c1198d79B9Ff61cD97C3bAca7F071E0,331
TDlabs,0x92aAe14a531086A665DAFF6eEdb5B71E00aCC6d1,166373688451407537492,1,0x8a12caB622B8093208931fA008D12D6Ba5AF47E4,331
Staked,0x2632A1CE82Fc88f4e0BfcEc1F19C6CF60c2e2254,166373685837011396745,1,0x8a12caB622B8093208931fA008D12D6Ba5AF47E4,331
Libertango,0xf87a83a0FB9118102b7fE2F19B7940dE3D421932,156140303895328584564,1,0x98c62d86634bB0E3f818EFC550e2F33369Eae7F3,331
Conclave-I,0x80048b9A29F77C92cD4172dF27CeE4D7BA3a8d07,110304399289934001888,1,0xAF2b000bAed202fA7de0d9DCeB6F6612De348011,331
Zanshin-v0,0x474df04481f778b46Fc71204C72B6A8BE396F0FF,166296792607272803997,1,0x8eB004daD9397B8f23E1279905c584920000756D,331
AUDIT.one-val-1,0x7Ff6C32b2CFc9bC6abdDAe17056e6d81c516806b,0,0,0xc764293De02189e248a8040a3D1b04EeC599Aa31,331
TeroLabs2,0x0c7610F8F4dA332C77c9800012009CcfA48F23e4,56960562169201203316,0,0x0E912990472bfe5948B5b7333Ecf528D5dA887cF,331
StakeService.com,0x48aC52662CBF32f8FC0cd8c1122c7560b482c2AD,18391170064255772237,0,0xb35Be22BccB0dB9dC62967dcF15fEB97C20f854e,331
YYYYYYYYYYYYY1,0x2B66f23c2987597225D36872a52a3D3337386054,92427039040800930099,1,0xE075ba4b1dCAF75513118d7aA08A057c658842c9,331
dedatalabs-val-1,0x00751D4E4861aDFFddDFCa5eC27703797A3B9349,166373645026227745164,1,0x1D083BECFAac301B4B845d0E5B80A9Da5F6DE946,331

Validator Group rewards:
Groupname,Group,Grouppayment,Validator,Validatorscore,Epochnumber
Moonlet.io,0x8851F4852ce427191Dc8D9065d720619889e3260,18254855547081293299,0x399b3D48Fdcd668d274afc6F4be05475A325C6c6,0.9874980919549,331
dynamic,0xD018838832f6112dE72EE6CC9967C56B333a0d1C,18462234712596245255,0x3BEbD7Ed4cA0ce7BFd69fC6aeF31dDD21C460158,0.9987162870115,331
huglester,0x614B7654ba0cC6000ABe526779911b70C1F7125A,18473694151761998833,0x5e69Cca114a77AB0a5804108FAF0cD1E1C802A5e,0.9993361864285,331
nonce,0xC7d5409fEe80B3Ac37Dbc111664dC511a5982469,18480186023721462584,0x1819b44553Fa39A983D1c92238E1F18AeE30eF51,0.99968736484,331
Projecttent,0x3D451dd723797b3DE938C5B22412032B6452591A,18186943108054160148,0xCDd88f80ed52AfEc9D670f5a6Da8940EEe716f45,0.9838243623115,331
Kytzu Group,0xc64DF5Be250264bf2888591D87cdeB13BFADC501,18147632303375924167,0xbBC724bdf633287E8bC18aC45800e2eD13877a73,0.9816978407122,331
Figment Networks,0x01b2b83fDf26aFC3Ca7062C35Bc68c8DdE56dB04,0,0x0e47ea88480788289bDa37D8Ae0C2CD680623cfA,0.9995212561132,331
syncnode_group,0x82f8Bcf96f24BA60Ef041D192c7CE04C907E2fb8,0,0xD5Fb8A8E6E81BC77d7D807c05b5b8aEAe7c2a458,0.9991689103783,331
DSRV CeloWhale.com,0xF83C93ea360B66DDCD532960304948B1c10786a1,35513956851776254512,0x588eD7dEefF8DF7c00FF187C398936b101583ec7,0.9605653832332,331
Simply Staking,0xAcdf897493A6000dbe256791E8A2beCbb405FD4F,0,0xc2023E0D13F53d181f29B99D829CD75628337D15,0.9883797680449,331
Easy2Stake Group,0x061E9958028dcAa66fd8B255AD95194203b6c4Da,18352819609660888628,0x1Dd301fEaa72F4C07b3ca56ce0C27e98E66B77Bd,0.9927974669419,331
sparkpool-group,0xEEFCfDFc8F5CeD9799A13EcA58DE2ba7534eAB92,18476501458793481194,0xA87710B8DC870483a2bE526EBca9F5048720eDce,0.9994880479609,331
Polychain Labs,0x47b2dB6af05a55d42Ed0F3731735F9479ABF0673,184859652231387654327,0x1d511366FEB1ed330925B2D51645bb5c607EA6D4,0.9999999911647,331
Polychain Labs,0x47b2dB6af05a55d42Ed0F3731735F9479ABF0673,184839802243182373134,0x5282F82f82E9a74E186b8Ee18cc76808b78DC383,0.9998926124707,331
missionquorum-group,0xA432da0Ed5A2c15cbc681227cCec3b375908FdCB,157822434834000055495,0x39b01620ecdEB8347270ee78dC962b0EA583b57a,0.853741914661,331
Polychain Labs,0x47b2dB6af05a55d42Ed0F3731735F9479ABF0673,184859652231387654327,0xAB0C16BBd78D6b9F4503D81D859A3FFB64486F9A,0.9999999911647,331
Polychain Labs,0x47b2dB6af05a55d42Ed0F3731735F9479ABF0673,184757294765436205054,0xACF0E11ff39EaD797475a1694FCca94A0e94784F,0.99944628751,331
cnstnt.xyz,0x21FB4411FA5828344c2788aB07D4cc12a12571b9,184749584350704186855,0x8d1A5eF84252586f0f841a9fCF4E31f23c4c21E0,0.9994045779505,331
Polychain Labs,0x47b2dB6af05a55d42Ed0F3731735F9479ABF0673,184859580115547767627,0x1E554ac038114C0A69834Be928FB95e5aaF18950,0.9999996010534,331
,0x7dbB949346Aeda9233218831455d57Eb8344Af82,184859653863180790921,0x822D6b9BBA2A3E460f3aB45fAAED98993522A8F0,0.9999999999919,331
cnstnt.xyz,0x21FB4411FA5828344c2788aB07D4cc12a12571b9,184524044490930319801,0xDF9121AFFb220e9095C875a79c3D1fA3EBEE4105,0.998184518002,331
,0x7dbB949346Aeda9233218831455d57Eb8344Af82,184465516362729977021,0x5423c03551641336b3D5886f85F882FD2BD5DB7E,0.9978679095535,331
,0x7dbB949346Aeda9233218831455d57Eb8344Af82,184857244233287249455,0x6964A0C39FA7f63C59F7dC77aC1802f24E2649Aa,0.9999869650768,331
,0x9eB1D96B5f0bdD3114eA1461182c688178ae6c4c,184859651301192362045,0xDd431a27ADce5B957FE2FE877a65F0e6304D68a0,0.9999999861328,331
,0x9eB1D96B5f0bdD3114eA1461182c688178ae6c4c,184859652231387654327,0xdA71A321C75A0Fe76073384bC52f6D583EAec517,0.9999999911647,331
Cypher Core,0x580F29aB04b7ED4E5aA86584D38c9317d9CF0358,18382228861873256372,0x4e957bBC61AFDab63263fFdbAF201F6f596E3e2D,0.9943883631487,331
ChainLayer,0x7194DFE766a92308880A943fD70F31c8E7c50e66,18481559162855285771,0x4FC4EA624db2E4A1d6195a03744D505CbcD9431b,0.9997616449279,331
a16z,0x4dA92da1afbf103F2B52DD29326e34C98cA1E78c,172640919049741566031,0xb2728EDcc9c255C2D053Ea6Cae31cFd584B758D3,0.9339026414932,331
a16z,0x4dA92da1afbf103F2B52DD29326e34C98cA1E78c,171978313842106466046,0xFcE0779607246eb5121a5A4192D6AC4A758C217D,0.9303182725204,331
a16z,0x4dA92da1afbf103F2B52DD29326e34C98cA1E78c,170775114752368576134,0x394B846E0eCf83Ef350A2A7Ccb7Be0C6552e7cBC,0.9238095559639,331
Figment Networks,0x01b2b83fDf26aFC3Ca7062C35Bc68c8DdE56dB04,0,0x90b056B00952d71bC7866c067b49f5b062098ac9,0.9290068139296,331
Figment Networks,0x01b2b83fDf26aFC3Ca7062C35Bc68c8DdE56dB04,0,0xDdc60B465c204Aa7358cE4f009D2dBC65af5C4B4,0.922889601424,331
StakesStoneGroup,0x3DcF2ED8dC84a63FfD2bfDc3CDb2fA0B1aeAfE5c,18357394680589259404,0x5F2E4e239bCD86e1cf97538Ea7D5E2Ec639a3Bc4,0.9930449558251,331
Staking Fund,0xFaa0b40645Ee5E8D9AE38393e8cDF8e5baA71d13,0,0x7b3859147B1C173FfF7182E58A4Ec81C1d287224,0.9999999999919,331
Bison Trails,0xDaDBd6Cfb29b054aDC9c4c2Ef0F21f0BBdb44871,171357006856760725364,0x089B1654325CcE14fb3e2dABCa2803bc489e83e7,0.9269573066614,331
“Gossip”,0x8954661e743c60C966E2ff6002B514126bb1cFe2,18420573047583350579,0x2DF505a1f281cd3ba10baA093fa89909C75362b5,0.9964625954059,331
The Passive Trust,0xB87f2354E34B26ba6406Ac60EA99DCD8cd5e63Bf,73923384390418682338,0x29c9CA0186650063ba6fcaC3a952613B11d6bC36,0.9997230715975,331
gunray,0x5402172E972b31Fc9F0383F53f45823Ab5037379,18484700064105798906,0xe5F947368C986DFe7a025B9B35deE3e5AC3a330e,0.9999315522703,331
Tango,0x98c62d86634bB0E3f818EFC550e2F33369Eae7F3,17652761181623925603,0xeD32008dB382d078bABB0eB6F91Df0207d909899,0.9549277418071,331
Nodito Labs,0x2c2B0f71d59B546B2CAfd222696589c13C3c325C,46214881917933201647,0xaB6eeE9dA588d38B43B1d2ea404e824fA2bF382c,0.9999993173581,331
Anchorage,0x35aE10F412503aBcf9275133613E8df7f56E72Be,171418098941978884075,0x43fbd2B45Ef5C042F821D24E14933C570e05Ef78,0.927287784859,331
CoinList,0x6f8e6ff69935F000B09d182A4739342fAEaF0c77,162872535092777612312,0x10bCB8943B7BF9b33E4D510feB212bD20b1cE9B2,0.8810604785185,331
CoinList,0x6f8e6ff69935F000B09d182A4739342fAEaF0c77,183176380453695252318,0x88f243bc0f439549DFFFF3C8c60b82f298116C9d,0.9908943169816,331
Anchorage,0x35aE10F412503aBcf9275133613E8df7f56E72Be,173082151441067934223,0xadBE75cBD14f92BA2D233b6b81E80D4634fAbB2a,0.9362894921775,331
Anchorage,0x35aE10F412503aBcf9275133613E8df7f56E72Be,181827527911708403531,0x1F15d4210e34DfeeEc087C906b426509da1b01f1,0.9835976867338,331
CoinList,0x6f8e6ff69935F000B09d182A4739342fAEaF0c77,183229010445804906118,0xd6DCbBC16cE0fBc95866a881F8F1728C48181507,0.9911790194086,331
Manta Labs,0xbf55dF76204f00AcF296F76cF4Aaf86A866a5eb0,181555112848511952518,0xc627459BFc70cD7B6719d5b8030C3beA7825CA7e,0.9821240549407,331
Bitcandy 🍬,0x34649AdA2cB44D851a2103Feaa8922DedDABfc1c,184307613536030064941,0x02DC997790352E12fA90eb1A937dFeF4bF8Dbbc0,0.9970137327583,331
Bi23 Labs,0x07fa1874ad4655AD0C763a7876503509be11e29E,18452209305068099986,0x997490F08B9b99dB00657e2B1BbEc180Cf804A27,0.998173961668,331
Anchorage,0x35aE10F412503aBcf9275133613E8df7f56E72Be,181709619806050835441,0x112fF12927CD6f924d80b9Aba32E531733B602fF,0.9829598617504,331
Manta Labs,0xbf55dF76204f00AcF296F76cF4Aaf86A866a5eb0,139027928563814809810,0x6e5a2624Ae582c96c5A12Eb168afA532a85AE829,0.7520728599091,331
HashQuark,0xD19FB36B7F433fe13820767ef6d0E26FDbaB68CC,92222981334214132271,0x758d1000fB3d37F88cE3BA880882898a3B0ca3aB,0.9977621336641,331
Celo.Works,0x3C86b6A27a074c1c4CC904d8808A1C33078DB4E6,14071192893231786420,0x084efAedbD599eff54f00d1b57Fab2F792555e79,0.7611824754109,331
missionquorum-group,0xA432da0Ed5A2c15cbc681227cCec3b375908FdCB,183477933923520243562,0x9e8A03768c2711c22bcc7999359D6655B7Cdb710,0.9925255732537,331
missionquorum-group,0xA432da0Ed5A2c15cbc681227cCec3b375908FdCB,179653150295050686437,0xdb2e80840f40033761A5cdFa3B09fE24f95a4aaa,0.9718353709921,331
,0x89d5bd54C43dDd10905A030DE6Ff02EbB6c51654,183770390687156835521,0xcED2449cCd1aae8077eC0670224e663CE19C8Fc4,0.9941076208099,331
,0x89d5bd54C43dDd10905A030DE6Ff02EbB6c51654,184859653729915466450,0x8CC5De79fC6C21531577F7B3d62c8413FB3bd303,0.999999999271,331
Bison Trails,0xDaDBd6Cfb29b054aDC9c4c2Ef0F21f0BBdb44871,116519395984617997998,0x30e0bfC25Df7642ee19Ff4fbCbE9ddeA4D1935bc,0.6303127456352,331
Chorus One,0x81cef0668e15639D0b101bdc3067699309D73BED,13838176063395953731,0x5098A28bFa2Da3183E36C009c7D63093b66441a5,0.9981032113891,331
cnstnt.xyz,0x21FB4411FA5828344c2788aB07D4cc12a12571b9,184853146878417773719,0x020e468c252FF18261b1996D25F0FfF4F830Cf7a,0.9999648003979,331
Chainflow,0x249a6B2B260000b08f50A2480e2d703bAf02E8BE,126652244511951514572,0xbcc71f31a5987F10Afd29F3BE7119C05A14F5bcB,0.9787521186568,331
Blockdaemon,0xCc4B2Bcbbc9639ef1E91F47acfD12Bd131525e79,18331806733493564285,0x45d8170f30b0d408196Ab0e14AF47D77b18f0A38,0.9916607734705,331
Perfect Stake,0x6CF4bB9ff947610944c6D8E0E5Ea26B1dEa73196,18386124048853244154,0x249eF45381b29D9955b365C33F202127499B873f,0.9945990736471,331
Anchorage,0x35aE10F412503aBcf9275133613E8df7f56E72Be,184493709963461660798,0x48dE6c6668c24e5Bd113d5C46d813C50fE2204D1,0.9980204230963,331
Ryabina | Celomap.io,0x8493BD3De67AC341D4cC11531f96a1A2cdBf29aD,0,0x1AEF9bf15de73aDBA318FC8aEa3D2f45d120CC8A,0.9837255417922,331
Bison Trails,0xDaDBd6Cfb29b054aDC9c4c2Ef0F21f0BBdb44871,184263372804894842557,0xC3957f7848d6b09DCb29d784036B0A302555E75b,0.9967744121158,331
Bison Trails,0xDaDBd6Cfb29b054aDC9c4c2Ef0F21f0BBdb44871,184008677420470001437,0x31cAF587f96B7C6fC8f33eB07ad854A21Eec64D9,0.9953966350882,331
Nodito Labs,0x2c2B0f71d59B546B2CAfd222696589c13C3c325C,45268667848651979494,0xAb2fDf3F4802A542754915E04c33a9596B164F31,0.9795251024713,331
Tessellated Geometry Alliance,0x0339Df3FE4f5ccC864EAE8491E5c8AEc4611A631,0,0x44a7Af700ACa1B24ECdB818dF4D202772D606Cb4,0.9999752250439,331
SlavaMo,0x602B65795BCc64b2fb329AC004236E194f077158,182829192150366730930,0xB29438f402da218180bBa79fbc0fAdD3e9E151aa,0.9890161986574,331
The Celo Group,0x81AE1C73A326325216E25ff1af9EA3871195036E,9242979851546352444,0xFE38631E7E164D20754387F7165B9C5b427dA17B,0.9999996925573,331
nga_group,0xc24baeac0Fd189637112B7e33d22FfF2730aF993,91777150008929494170,0x4D540eC31F9CC5b1d10a329EB4Eddf52CB3dC000,0.9929386763443,331
Masternode24.de,0x95aE59515915D6c493a846aE022F93726652b50A,91832362736350164008,0x9C542c0B5aF5e04E2D1e67cf1fC176E91bC18dA0,0.9935360238592,331
AG,0x15Ed3f6b79F5Fb9Ef1D99D37314Dd626b3005F0b,36921118732477335972,0x943e9a5c750adcc467575F4fE81bf6b1C3A2aeD7,0.998625658996,331
moonli.me,0xc8A81D473992c7c6D3F469A8263F24914625709d,0,0xDdbca691842FD452d8F53575FdFdC2314eb3AD57,0.9999999999919,331
Alive29,0x0f66619058BB9675f3d394FCc2cE236a29901571,91390085124185585530,0x28B5f40C3f5883021408Ce79F20FCf282E873236,0.9887510142271,331
chainvibes,0xD72Ed2e3db984bAC3bB351FE652200dE527eFfcf,92428884682358009852,0x00A9e8869e40e038D0FaE63D825a732e6FfF48d1,0.9999898057801,331
Virtual Hive,0x71dcC67baEECd9308e341359f70B782D9C3565b8,18467170273398016185,0xe51662a4448cc9994Df7B28E4F0b38D678958B0D,0.9989832766276,331
Manta Labs,0xbf55dF76204f00AcF296F76cF4Aaf86A866a5eb0,116803763012807766290,0xaC6492D439b3b812e822d1e975338f48bDD0B1f2,0.631851031693,331
HappyCelo,0x481EAdE762d6D0b49580189B78709c9347b395bf,0,0xF0b25502595911b64de75F453b34557C1b2c4760,0.9991916709238,331
Censusworks,0x0d4f8cEA48cDaDEAe345431577a64983c0535B12,184469415341765622074,0x2bDc5CcdA08A7F821ae0Df72B5Fda60Cd58d6353,0.9978890011165,331
Validator.Capital,0xB33e9e01E561a1Da60f7Cb42508500e571afb6Eb,184859444392883917758,0x0F412760759a2fAD4f07ceE36bfdAa218814DE89,0.9999988668604,331
wotrust,0x7C75B0B81A54359E9dCCDa9cb663ca2e3De6B710,184859646632081168593,0x6ac7E3D0BDeb78ddcF3d9b15208B50C08271C213,0.9999999608752,331
keyko.io,0x2fd49E97262D505Fd76BB6E0e06eC10e1fd54589,18485963229050284808,0x233c06EDc757003cA4ec078012F517F21A24C55c,0.9999998832943,331
PretoriaResearchLab.io,0xddaa60B6d803674bBc434F1C2B261CeB67C2fd7c,18485965377932845192,0xe4036C56B0026eb86f058c77c55E73e9d65d2EA0,0.9999999995383,331
Usopp | usopp.club,0x067e453918f2c44D937b05a7eE9DBFB804C54ADd,91846762011568475265,0x5F49e61f127D315ECDE1702430EC991DAaA9C3bF,0.993691809883,331
Goods & Services,0x8580dB53C3ebC56230662B771ceF2707E92Ef83A,18485959867586370687,0xF2aADf8fF2Ad7e5C1efb0d45Cb622618BA6636A7,0.9999997014556,331
celvaly,0xd25c6a9FEf4744E8d4F90Bf6bdFAF7686909d799,18485477527120892288,0x989F3F2684F96B8cDeC308B4E7538A5a062890f0,0.9999736092037,331
wotrust,0x7C75B0B81A54359E9dCCDa9cb663ca2e3De6B710,184859653705957655309,0x206296BeffFB68fCE658535D443c732892162005,0.9999999991414,331
swiftstaking,0xE141831c2c1198d79B9Ff61cD97C3bAca7F071E0,0,0x77eCf3c7AC7082344Edfee10e41Edb72fA3D1951,0.9806877625744,331
TDlabs | Staked.us,0x8a12caB622B8093208931fA008D12D6Ba5AF47E4,18485965383489726388,0x92aAe14a531086A665DAFF6eEdb5B71E00aCC6d1,0.9999999998389,331
TDlabs | Staked.us,0x8a12caB622B8093208931fA008D12D6Ba5AF47E4,18485965093001266305,0x2632A1CE82Fc88f4e0BfcEc1F19C6CF60c2e2254,0.9999999841249,331
Tango,0x98c62d86634bB0E3f818EFC550e2F33369Eae7F3,17348922655036509396,0xf87a83a0FB9118102b7fE2F19B7940dE3D421932,0.9384915687301,331
Conclave,0xAF2b000bAed202fA7de0d9DCeB6F6612De348011,73536266193289334592,0x80048b9A29F77C92cD4172dF27CeE4D7BA3a8d07,0.9944877729664,331
ZanshinDojo.org,0x8eB004daD9397B8f23E1279905c584920000756D,18477421400808089333,0x474df04481f778b46Fc71204C72B6A8BE396F0FF,0.9995378123089,331
AUDIT.one,0xc764293De02189e248a8040a3D1b04EeC599Aa31,63573234964062817201,0x7Ff6C32b2CFc9bC6abdDAe17056e6d81c516806b,0.3439,331
TeroLabs,0x0E912990472bfe5948B5b7333Ecf528D5dA887cF,6328951352133467035,0x0c7610F8F4dA332C77c9800012009CcfA48F23e4,0.3423652062427,331
StakeService.com,0xb35Be22BccB0dB9dC62967dcF15fEB97C20f854e,18391170064255772236,0x48aC52662CBF32f8FC0cd8c1122c7560b482c2AD,0.198974407663,331
YYYYYYYYYYYYY,0xE075ba4b1dCAF75513118d7aA08A057c658842c9,92427039040800930095,0x2B66f23c2987597225D36872a52a3D3337386054,0.9999698377501,331
deDataLabs,0x1D083BECFAac301B4B845d0E5B80A9Da5F6DE946,18485960558469749462,0x00751D4E4861aDFFddDFCa5eC27703797A3B9349,0.999999738829,331

thanks